Say/ViewController.swift

```diff
@@ -14,14 +14,14 @@ import Cocoa
 class MainWindow: NSWindow {
     @IBOutlet var speechToolbarItem: NSToolbarItem! = nil;
     @IBOutlet var exportToolbarItem: NSToolbarItem! = nil;
-
+    @IBOutlet var importToolbarItem: NSToolbarItem! = nil;
```

Comment:
One more stuff: Swift doesn't require semicolon end of the statement. Will you remove it too? (It is fault of the original code)